src/expression.rs

```diff
@@ -45,13 +45,30 @@ impl<'src> Expression<'src> {
   pub(crate) fn variables<'expression>(&'expression self) -> Variables<'expression, 'src> {
     Variables::new(self)
   }
+
+  pub(crate) fn empty_string_literal() -> Self {
```

Comment:
Creating empty string literals should be avoided.

Reply:
I think it's less clear to create empty string literals, rather than to handle the none case separately.